app/javascript/components/miq-ae-customization/index.jsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState, useEffect } from 'react';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import { Button } from 'carbon-components-react';
+import MiqFormRenderer, { useFormApi } from '@@ddf';
+import PropTypes from 'prop-types';
+import createSchema from './miq-ae-customization-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const MiqAeCustomization = ({ dialogRecord, dialogTypes }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [initialValues, setInitialValues] = useState(undefined);
```

Comment:
It's always better to put them in a single object. We can avoide multiple renders if we put them all in an object like

 ```
 const [data, setData] = useState({
    isLoading: true,
    initialValues: undefined,
  });
```